packages/usa-radio/src/styles/_usa-radio.scss

```diff
@@ -71,17 +62,15 @@
     units($theme-input-select-size);
   position: relative;
 
-  &:before {
+  &::before {
     content: " ";
     display: block;
     left: 0;
     margin-left: units($theme-input-select-border-width);
-    margin-top: math.div(
-      (
-        line-height($theme-form-font-family, $theme-input-line-height) *
-          font-size($theme-form-font-family, $theme-body-font-size) -
-          units($theme-input-select-size)
-      ),
+    margin-top: divide(
```

Comment:
@mejiaj Should this `divide` be `math.div`?

Reply:
It should! Fixed in 38039959d9672eaacb8546f8a3d579087b9d4e3e.